examples/deck/example.js

```diff
@@ -143,7 +143,9 @@ export default class Example extends Component<
   }
 
   _renderAllCheckboxes() {
-    const { testFeatures: { features } } = this.state;
+    const {
+      testFeatures: { features }
+    } = this.state;
```

Comment:
I have not changed this. A similar thing happened in some of our other repos, it was caused by `jsprettier` version not being pinned in `package.json`. 

If you ran tests on this it would probably revert based on your installed version.

What do you think, should we pin it to a specific version?